### PR TITLE
feat: added support for mssql v11

### PR DESCRIPTION
**Pull request by instanacd**
refs https://jsw.ibm.com/browse/INSTA-8736

### CODE REVIEW

**Review by aryamohanan — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -182,8 +182,8 @@
     "mongoose-v6": "npm:mongoose@^6.12.7",
     "mongoose-v7": "npm:mongoose@^7.6.10",
     "morgan": "^1.10.0",
-    "mssql": "^10.0.3",
-    "mssql-v9": "npm:mssql@^9.0.1",
```
Comment (kirrg001) [inline package.json:186]: Suggestion: Only run v11 tests.
If there is no "real change" between major versions, there is no need to run both versions.

Comment (kirrg001) [reply package.json:186]: I just noticed that the adapted test is anyway wrong. You are just iterating over the versions but you are not using both versions (mock missing).

**Review by kirrg001 — CHANGES_REQUESTED** [review]
See second comment

File: package.json (hunk 1 above)
Comment (kirrg001) [reply package.json:186]: My bad. The logic existed already 👍

**Comment by aryamohanan** [comment]
> LGTM
> 
> Please consider my suggestion: https://github.com/instana/nodejs/pull/1190/files#r1647083061
> 
> Otherwise, feel free to merge

Considering that we still support node versions 16 and 14, and the latest MSSQL doesn't support these versions, I intend to retain the tests for versions 10 and 11.

**Pull request merged**